MainGame.py

```diff
@@ -0,0 +1,57 @@
+from Player import Player
+
+class MainGame:
+    def __init__(self):
+        self.player_one = Player(self.get_user_name(1), self.get_user_symbol(1))
+        self.player_two = Player(self.get_user_name(2), self.get_user_symbol(2))
+
+    def get_user_name(self, player_count) -> str:
+        new_player_name = input(f"Player {player_count}: Please enter your name: ")
+
+        return new_player_name
+
+    def get_user_symbol(self, player_count) -> str:
+        while True:
+            try:
+                new_player_symbol = input(
+                    f"Player {player_count}: Please enter your wanted symbol: "
+                )
+                if new_player_symbol == "X" or new_player_symbol == "O":
```

Comment:
```suggestion
                if new_player_symbol == "X" or new_player_symbol == "O":
                
```

Reply:
It is just a return statement in an if clause, I don't think we have to add in a new line there